scripts/install_runtime_dependencies.sh

```diff
@@ -1,22 +1,13 @@
-#!/bin/bash
+#!/bin/sh
+set -eu
 
 apt-get update
-apt-get install -y curl libdlib19.1 ffmpeg exiftool libheif1
-
-# Install Darktable if building for a supported architecture
-if [ "${TARGETPLATFORM}" = "linux/amd64" ] || [ "${TARGETPLATFORM}" = "linux/arm64" ]; then
-  echo 'deb [trusted=true] https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/ /' > /etc/apt/sources.list.d/darktable.list
-  # Release key is invalid, just trust the repo
-  curl -fsSL https://download.opensuse.org/repositories/graphics:/darktable/Debian_12/Release.key \
-    | gpg --dearmor -o /etc/apt/trusted.gpg.d/darktable.gpg
-  gpg --show-keys --with-fingerprint --dry-run /etc/apt/trusted.gpg.d/darktable.gpg
-
-  apt-get update
-  apt-get install -y darktable
-fi
-
-# Remove build dependencies and cleanup
-apt-get purge -y ${BUILD_DEPENDS[@]}
-apt-get autoremove -y
-apt-get clean
-rm -rf /var/lib/apt/lists/*
+apt-get install -y curl exiftool
+
+# libheif dependencies: no dependency
+# libraw dependencies
+apt-get install -y libjpeg62-turbo liblcms2-2
+# ImageMagick dependencies
+apt-get install -y libjbig0 libtiff6 libfreetype6 libjxl0.7 liblqr-1-0 libpng16-16 libdjvulibre21 libwebpmux3 libwebpdemux2 libwebp7 libopenexr-3-1-30 libopenjp2-7 libjpeg62-turbo liblcms2-2 libxml2 libx11-6 libgomp1
+# go-face dependencies
+apt-get install -y libdlib19.1 libblas3 liblapack3 libjpeg62-turbo
```

Comment:
I want to separate dependencies for different libraries. It won't reinstall the package if it's already done, so calling multiple times won't increase much time.

A multiline command is sometimes used to avoid a line too long. I tried that before and I found using multiple `apt-get` is clearer than that.

All packages listed here are the direct-dependencies of each library. If we remove/switch any packages in the future, just removing the correspond line.

Reply:
Ok, so I guess that this form still meets your goals:
```bash
apt-get install -y \
  curl exiftool \
  # libheif dependencies: no dependency
  # libraw dependencies
  libjpeg62-turbo liblcms2-2 \
  # ImageMagick dependencies
  libjbig0 libtiff6 libfreetype6 libjxl0.7 liblqr-1-0 libpng16-16 libdjvulibre21 libwebpmux3 libwebpdemux2 libwebp7 \
  libopenexr-3-1-30 libopenjp2-7 libjpeg62-turbo liblcms2-2 libxml2 libx11-6 libgomp1 \
  # go-face dependencies
  libdlib19.1 libblas3 liblapack3 libjpeg62-turbo
```
What do you think?

Reply:
Well, why not. Updated.

Reply:
Now we know that command doesn't work, see [here](https://github.com/photoview/photoview/actions/runs/10899558813/job/30245223797?pr=1053#step:5:1677).

Reverted.

Reply:
`shell`/`bash` doesn't support mix commands and comments in a multi-line command.

It works with `Dockerfile` because Docker parses the file in a wrong way. Docker removes all comment lines first, then calls the `RUN` command with a shell.

Reply:
Oh, ok. Then, sure, let's keep them as is...
Very strange... I was sure that it works, as I used it many times in Dockerfiles and CI YAML files...

Reply:
Both Dockerfile and YAML files are parsed with two steps, first is Dockerfile or YAML format, which erases all comments in the file, then put commands to a shell. It's not the same way as shell parsing scripts.